network/ServerNetworking.cpp

```diff
@@ -485,7 +485,7 @@ void ServerNetworking::SetHostPlayerID(int host_player_id)
 { m_host_player_id = host_player_id; }
 
 void ServerNetworking::Init() {
-    tcp::endpoint endpoint(tcp::v4(), Networking::MessagePort());
+    tcp::endpoint endpoint(tcp::v6(), Networking::MessagePort());
```

Comment:
In that case, a comment on the line, and any similar, indicating that this supports both v4 and v6 would be helpful.